ardupilot/equipment/power/20010.BatteryContinuous.uavcan

```diff
@@ -0,0 +1,35 @@
+#
+# Battery data to be sent continuously
+#
+
+float16 temperature_cells       # [C]      : Pack mounted thermistor (preferably installed between cells), NAN: field not provided
+float16 temperature_pcb         # [C]      : Battery PCB temperature (likely output FET(s) or current sense resistor), NAN: field not provided
+float16 temperature_other       # [C]      : Application dependent, NAN: field not provided
+float32 current                 # [Ampere] : Positive: defined as a discharge current. Negative: defined as a charging current, NAN: field not provided
+float16 voltage                 # [Volt]
+uint8 state_of_charge           # [%]      : The estimated state of charge as a percent remaining between 0 and 100.
+uint8 slot_id                   #          : The physical location of the battery on the aircraft. 0: field not provided
+uint32 capacity_consumed        # [mAh]    : This is either the consumption since power-on or since the battery was full, depending on the value of STATUS_FLAG_CAPACITY_RELATIVE_TO_FULL, UINT32_MAX: field not provided
+uint32 capacity_remaining       # [mAh]    : If STATUS_FLAG_CAPACITY_RELATIVE_TO_FULL is unset, this value is based on the assumption the battery was full when the system was powered, UINT32_MAX: field not provided
+uint32 full_charge_capacity     # [mAh]    : Predicted battery capacity when fully charged (accounting for battery degradation), UINT32_MAX: field not provided
```

Comment:
@hamishwillee @hendjoshsr71 integer types were chosen here to match mavlink, but the [Battery Status V2](https://github.com/mavlink/mavlink/pull/1846/files#diff-413c1cf6a533671c65dda0eeaa0280a129cbf251a06d3ad7ef9ea2bbdb34f1f6R421-R422) has since changed to floats. I am going to change these here as well to match

Reply:
Sounds good! Thanks !